src/scanForInputs.js

```diff
@@ -11,7 +11,7 @@ const _forms = new Map()
  * the synchronous mutations via findEligibleInputs
  *
  * @param DeviceInterface
- * @param {Map<HTMLFormElement, Form>} [forms]
+ * @param {Map<HTMLElement, Form>} [forms]
```

Comment:
These are not necessarily `form` elements, because sometimes no formal `form` is defined and we try to infer a container element. This type is more accurate.